Java.gitignore

```diff
@@ -22,3 +22,12 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 replay_pid*
+
+# Gradle specific files
+.gradle/
+build/
```

Comment:
See the comment about the eclipse specific template. I would prefer we create a new Global entry for Gradle.

In addition, we could also consider adding a descriptive comment and leaving these entries, but having them commented out by default.
[Example of this in the Python template.](https://github.com/github/gitignore/blob/main/Python.gitignore#L157-L162)